scene/gui/text_edit.h

```diff
@@ -353,10 +356,18 @@ class TextEdit : public Control {
 	PopupMenu *menu = nullptr;
 	PopupMenu *menu_dir = nullptr;
 	PopupMenu *menu_ctl = nullptr;
+	PopupPanel *color_popup = nullptr;
+	ColorPicker *color_picker = nullptr;
```

Comment:
Is it possible to move the ColorPicker to ScriptTextEditor? The TextEdit doesn't use it on its own.